### ISSUE TITLE
stashcp package fails to install on Ubuntu 22.04

### ISSUE THREAD

**Issue by matyasselmeci** [htcondor/osdf-client#29]
```
% sudo dpkg -i ./stashcp-6.9.4-1_amd64.deb

dpkg: error processing archive ./stashcp-6.9.4-1_amd64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 8 package 'stashcp':
 'Replaces' field, syntax error after reference to package 'stashcache-client'
Errors were encountered while processing:
 ./stashcp-6.9.4-1_amd64.deb
```

**Comment by djw8605** [comment]
Is this fixed with #31 ?

**Comment by matyasselmeci** [comment]
Yep.
